common4j/src/main/com/microsoft/identity/common/java/nativeauth/commands/parameters/AcquireTokenNoFixedScopesCommandParameters.java

```diff
@@ -57,16 +59,27 @@ public class AcquireTokenNoFixedScopesCommandParameters extends BaseNativeAuthCo
 
     private final IAccountRecord account;
 
-    @Expose()
     @NonNull
     private final AbstractAuthenticationScheme authenticationScheme;
 
-    @Expose()
     private final boolean forceRefresh;
 
-    private final String loginHint;
+    @NotNull
+    @Override
+    public String toUnsanitizedString() {
+        return "AcquireTokenNoFixedScopesCommandParameters(account=" + account + ", authenticationScheme=" + getAuthenticationScheme() + ", forceRefresh=" + forceRefresh + ", authority=" + authority + ", challengeTypes=" + challengeType + ")";
+    }
 
-    private final List<Map.Entry<String, String>> extraOptions;
+    @Override
+    public boolean containsPii() {
+        return !toString().equals(toUnsanitizedString());
+    }
```

Comment:
This implementation can probably move to `ILoggable` itself and then you won't have to override it everywhere

Reply:
Looks you have defined it there already so not sure why there is a need to override in so many places

Reply:
Java doesn't play nice with Kotlin (because it's a Kotlin interface with a default function? I'm not entirely sure tbh). 
1. not overriding the method at all gives compilation error `Class 'AcquireTokenNoFixedScopesCommandParameters' must either be declared abstract or implement abstract method 'containsPii()' in 'ILoggable'`
2. overriding it and performing `return super.containsPii();` gives build error `error: abstract method containsPii() in ILoggable cannot be accessed directly`.